pkg/services/sqlstore/migrations/external_alertmanagers.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func AddExternalAlertmanagerToDatasourceMigration(mg *migrator.Migrator) {
-	mg.AddMigration("migrate external alertmanagers to datsourcse", &externalAlertmanagerToDatasources{})
```

Comment:
Is renaming a migration safe or it could cause creating a new migration? 🤔 cc @grafana/grafana-search-and-storage @grafana/backend-platform